### ISSUE TITLE
VS Code Debug. Debug Console autocomplete errors.

### ISSUE THREAD

**Issue by smokeelow** [oracle/graal#1227]
### Issue
I'm using VS Code to debug JavaScript. In debug, when i'm trying to type something in _Debug Console_
i'm getting error

```
Error processing "completions": TypeError: Cannot read property 'length' of undefined
    at NodeDebugAdapter.getFlatAndUniqueCompletionItems (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/ms-vscode.node-debug2/node_modules/vscode-chrome-debug-core/out/src/chrome/chromeDebugAdapter.js:3202:32)
    at NodeDebugAdapter.<anonymous> (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/ms-vscode.node-debug2/node_modules/vscode-chrome-debug-core/out/src/chrome/chromeDebugAdapter.js:3193:27)
    at Generator.next (<anonymous>)
    at fulfilled (/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/ms-vscode.node-debug2/node_modules/vscode-chrome-debug-core/out/src/chrome/chromeDebugAdapter.js:11:26)
```

See full video https://www.youtube.com/watch?v=VINnHJqmC60&feature=youtu.be

When i go by url chrome-devtools://devtools/bundled/js_app.html?ws=127.0.0.1:7992/3ad2426f-e6a244b0f3a in Chrome, it works as expected. I see autocomplete, no errors, everything works perfect.

In Intellij IDEA/WebStorm i don't see errors but autocomplete not working in debug console.

My setup:
_OS:_ macOS 10.14
_Editor:_ VS Code Version 1.33.1 (1.33.1) (autocomplete not working, errors)
_Editor:_ Chrome(DevTools) 74 (autocomplete works, no errors)
_Editor:_ Intellij IDEA 2019.1 (autocomplete not working, no errors)
_JVM:_ GraalVM RC16

**Comment by smokeelow** [comment]
Reproduced in Graal 19.1

**Comment by smokeelow** [comment]
Reproduced in Graal 19.2.1

**Comment by entlicher** [comment]
Reproduced, fix is in progress.

**Comment by entlicher** [comment]
Fixed in the upcoming 19.3.
https://github.com/oracle/graal/commit/5b5948895ab46e57b90f54b0da9310f18e8db7c0

**Comment by smokeelow** [comment]
@entlicher great, thank you!
